cluster-autoscaler/utils/kubernetes/listers.go

```diff
@@ -49,41 +49,41 @@ type ListerRegistry interface {
-	daemonSetLister             v1appslister.DaemonSetLister
-	replicationControllerLister v1lister.ReplicationControllerLister
-	jobLister                   v1batchlister.JobLister
-	replicaSetLister            v1appslister.ReplicaSetLister
-	statefulSetLister           v1appslister.StatefulSetLister
+	allNodeLister                      NodeLister
+	readyNodeLister                    NodeLister
+	scheduledPodLister                 PodLister
+	scheduledAndUnschedulablePodLister PodLister
+	podDisruptionBudgetLister          PodDisruptionBudgetLister
+	daemonSetLister                    v1appslister.DaemonSetLister
+	replicationControllerLister        v1lister.ReplicationControllerLister
+	jobLister                          v1batchlister.JobLister
+	replicaSetLister                   v1appslister.ReplicaSetLister
+	statefulSetLister                  v1appslister.StatefulSetLister
 }
 
 // NewListerRegistry returns a registry providing various listers to list pods or nodes matching conditions
 func NewListerRegistry(allNode NodeLister, readyNode NodeLister, scheduledPod PodLister,
-	unschedulablePod PodLister, podDisruptionBudgetLister PodDisruptionBudgetLister,
+	scheduledAndUnschedulablePodLister PodLister, podDisruptionBudgetLister PodDisruptionBudgetLister,
 	daemonSetLister v1appslister.DaemonSetLister, replicationControllerLister v1lister.ReplicationControllerLister,
 	jobLister v1batchlister.JobLister, replicaSetLister v1appslister.ReplicaSetLister,
 	statefulSetLister v1appslister.StatefulSetLister) ListerRegistry {
 	return listerRegistryImpl{
-		allNodeLister:               allNode,
-		readyNodeLister:             readyNode,
-		scheduledPodLister:          scheduledPod,
-		unschedulablePodLister:      unschedulablePod,
-		podDisruptionBudgetLister:   podDisruptionBudgetLister,
-		daemonSetLister:             daemonSetLister,
-		replicationControllerLister: replicationControllerLister,
-		jobLister:                   jobLister,
-		replicaSetLister:            replicaSetLister,
-		statefulSetLister:           statefulSetLister,
+		allNodeLister:                      allNode,
+		readyNodeLister:                    readyNode,
+		scheduledPodLister:                 scheduledPod,
+		scheduledAndUnschedulablePodLister: scheduledAndUnschedulablePodLister,
+		podDisruptionBudgetLister:          podDisruptionBudgetLister,
+		daemonSetLister:                    daemonSetLister,
+		replicationControllerLister:        replicationControllerLister,
+		jobLister:                          jobLister,
+		replicaSetLister:                   replicaSetLister,
+		statefulSetLister:                  statefulSetLister,
 	}
 }
 
 // NewListerRegistryWithDefaultListers returns a registry filled with listers of the default implementations
 func NewListerRegistryWithDefaultListers(kubeClient client.Interface, stopChannel <-chan struct{}) ListerRegistry {
-	unschedulablePodLister := NewUnschedulablePodLister(kubeClient, stopChannel)
```

Comment:
`NewUnschedulablePodLister`, `NewUnschedulablePodInNamespaceLister` and `UnschedulablePodLister` are no longer used anywhere. Could you remove them? We don't want to keep dead code.

Reply:
Done